core/src/test/scala/akka/persistence/jdbc/SingleActorSystemPerTestSpec.scala

```diff
@@ -70,6 +76,8 @@ abstract class SingleActorSystemPerTestSpec(val config: Config)
 
   def withActorSystem(f: ActorSystem => Unit): Unit = {
     implicit val system: ActorSystem = ActorSystem("test", config)
+    // start new schema on each system creation and for each individual test
+    Await.ready(SchemaUtils.dropAndCreate(system), 2.seconds)
```

Comment:
```suggestion
    SchemaUtils.dropAndCreate(system).futureValue
```

use `futureValue` from `org.scalatest.concurrent.ScalaFutures`?

Reply:
Ah, just noticed @johanandren's comment about patience config.